pom.xml

```diff
@@ -1298,7 +1299,7 @@
         <plugin>
           <groupId>org.codehaus.mojo</groupId>
           <artifactId>findbugs-maven-plugin</artifactId>
-          <version>${findbugs.version}</version>
```

Comment:
revert? or remove `findbugs.version` property if this is the only place it's used?